Variable Set Permission Update (#2238)
hashicorp/terraform-website: content/cloud-docs/workspaces/variables/managing-variables.mdx

> **Hands On:** Try the [Manage Variable Sets in Terraform Cloud tutorial](https://learn.hashicorp.com/tutorials/terraform/cloud-multiple-variable-sets) on HashiCorp Learn.

> Variable Sets can only be managed by members of the [Owners Team](/cloud-docs/users-teams-organizations/permissions#organization-owners) or members of a team with the [Manage Workspaces](/cloud-docs/users-teams-organizations/permissions#manage-workspaces) permission assigned.

Reviewer: Just want to update it to active voice :-) 

```suggestion
Only members of the [Owners Team](/cloud-docs/users-teams-organizations/permissions#organization-owners) or members of a team with the [Manage Workspaces](/cloud-docs/users-teams-organizations/permissions#manage-workspaces) permission can create, update, and delete variable sets.
```

> Variable sets are only visible to members with the correct team allocation. This is members who are part of the [Owners Team](/cloud-docs/users-teams-organizations/permissions#organization-owners) or members of a team with [Manage Workspaces](/cloud-docs/users-teams-organizations/permissions#manage-workspaces) permissions assigned.

Reviewer 2: ```suggestion
Variable Sets can only be managed by members of the [Owners Team](/cloud-docs/users-teams-organizations/permissions#organization-owners) or members of a team with [Manage Workspaces](/cloud-docs/users-teams-organizations/permissions#manage-workspaces) permissions assigned.
```
